bin/cylc-run

```diff
@@ -95,7 +96,11 @@ warm start unless you use special cold-start tasks. See "Suite Start-up" and
             start_point_string = self.args[1]
             if start_point_string == "now":
                 start_point_string = (
-                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%MZ")
+                    datetime.datetime.now().strftime("%Y%m%dT%H%M")
```

Comment:
I figured minutes required there in case they're needed, but if they're not needed the truncated suite cycle point format will (in effect) ignore them?   If not, I'll consider your syntax tomorrow (I'm getting too sleepy here tonight).